### PR TITLE
fixed linter issues

### PR DESCRIPTION
**Pull request by bruceaxoniq**
fixed contractions and passive style

### CODE REVIEW

File: playbook-dev.yaml
Comment (azzazzel) [inline playbook-dev.yaml:1]: Let's keep the `playbook-dev.yaml` as a reference in the repo. 
If you need a custom build just copy it under some other name and use that locally instead.

File: playbook-dev.yaml [hunk 1]
```diff
@@ -7,13 +7,13 @@ content:
...
-    branches: [development, 'synapse-*']
-  - url: .
-    start_paths: ['localLinks/giftcard-demo/docs/*', '!localLinks/giftcard-demo/docs/_playbook']
+ # - url: .
+ #   start_paths: ['localLinks/AxonFramework/docs/*', '!localLinks/AxonFramework/docs/_playbook']
+  #- url: ./localLinks/AxonSynapse
+  #  start_paths: ['/docs/*', '!/docs/_playbook']
+  #  branches: [development, 'synapse-*']
+  #- url: .
+  #  start_paths: ['localLinks/giftcard-demo/docs/*', '!localLinks/giftcard-demo/docs/_playbook']
```
Comment (azzazzel) [inline playbook-dev.yaml:16]: I guess you know that, but by commenting those out, you build only the content in this repo, and thus, that is the only content the prose linter is checking. 

You really need those to check the errors in the other repos.

File: vale/Vocab/Names_Terms/accept.txt
Comment (azzazzel) [inline vale/Vocab/Names_Terms/accept.txt:1]: I added those to the Vale rules. This file will be overwritten by the production build every time as we have `update_styles: true` there.

**Review by azzazzel — COMMENTED** [review]
@bruceaxoniq Everything looks good except for the playbook and Vale terms changes.

**Review by azzazzel — CHANGES_REQUESTED** [review]
@bruceaxoniq  can you pleas update the PR according to the previous comments

**Comment by bruceaxoniq** [comment]
@azzazzel I updated the PR to remove the files that were supposed to stay local on my machine.

**Review by azzazzel — APPROVED** [review]

**Pull request merged**